README.md

```diff
@@ -1,27 +1,38 @@
-The [Dart](https://www.dart.dev/) implementation of
-[gRPC](https://grpc.io/): A high performance, open source, general RPC framework that puts mobile and HTTP/2 first.
+# gRPC Dart
```

Comment:
Remove title – this is redundant to the title on the GitHub page and package page

Reply:
Many (if not most) pages on GH use a first-level heading as a title, e.g.:

- https://pub.dev/packages/analyzer -> [README](https://github.com/dart-lang/sdk/blob/master/pkg/analyzer/README.md)
- https://pub.dev/packages/crypto -> [README](https://github.com/dart-lang/crypto/blob/master/README.md)
- https://pub.dev/packages/meta -> [README](https://github.com/dart-lang/sdk/blob/master/pkg/meta/README.md)
- https://github.com/dart-lang/protobuf -> [README](https://github.com/dart-lang/protobuf/blob/master/README.md)

To list a few, which are Dart packages. And all grpc-* repos use a title.

Reply:
Right. But imagine how this will look at https://pub.dev/packages/grpc

Double titles!

Reply:
But those titles serve difference purposes IMHO.

Ok, your call.

Reply:
Done